src/css/override_components.scss

```diff
@@ -1,6 +1,12 @@
 
 @import 'override_vars.scss';
 
+
+.usa-footer-primary-section {
+  background-color: $color-background-light;
+}
+
+
```

Comment:
This fixes the slight color differences between the body background and footer background. 

**Screenshot is of the two different colors.**

![screen shot 2017-01-05 at 10 15 55 am](https://cloud.githubusercontent.com/assets/1449852/21685858/2808bb7c-d331-11e6-8bba-2a7f21459dbe.png)

**Background color fixed**

![screen shot 2017-01-05 at 10 25 02 am](https://cloud.githubusercontent.com/assets/1449852/21685901/4fb975b2-d331-11e6-8a01-a7778acf0974.png)